roles/ceph-common/tasks/main.yml

```diff
@@ -93,10 +88,24 @@
   # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
   static: False
 
-- include: facts.yml
-- include: ./checks/check_socket.yml
+- name: get ceph version
```

Comment:
@dillaman this task was already executed in `ceph-common` before this refactor. Do you mean before this commit you didn't hit this issue?